style.css

```diff
@@ -1,80 +1,636 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable declaration-block-no-shorthand-property-overrides */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable color-hex-case */
 /* stylelint-disable font-family-no-missing-generic-family-keyword */
```

Comment:
- [ ] Linters are nice tools that ensure that we write our codes in a uniform and easily understood way, especially because we will be working with other developers. This why managing and understanding some industry standards is best for you. Kindly remove all these and find ways to resolve the lining issues you might be facing. :pray: